src/utils/navigation.ts

```diff
@@ -42,7 +51,7 @@ export const NAVIGATION: Navigation = {
   },
   [config.OVERSEAS_ENTITY_QUERY_URL]: {
     currentPage: config.OVERSEAS_ENTITY_QUERY_PAGE,
-    previousPage: () => config.UPDATE_INTERRUPT_CARD_URL,
+    previousPage: (appData: ApplicationData, req: Request) => getRemoveBackLink(req, config.UPDATE_INTERRUPT_CARD_URL),
```

Comment:
i don't think it should be called getRemoveBackLink as it is not just for remove, so how about just getBackLink(...)

Reply:
also, do you need to include appData? it doesn't get passed to getRemoveBackLink

Reply:
True. We wouldn't call this for a 'ROE registration' screen though. Should it maybe be called `getUpdateRemoveBackLink`?

Reply:
sounds good, could have an 'or' in there - getUpdateOrRemoveBackLink

Reply:
> also, do you need to include appData? it doesn't get passed to getRemoveBackLink

Yes, because it's an optional param. This question has cropped up before, but yes, needed.